images/linux/ubuntu2004.json

```diff
@@ -251,7 +251,8 @@
                 "{{template_dir}}/scripts/installers/yq.sh",
                 "{{template_dir}}/scripts/installers/pypy.sh",
                 "{{template_dir}}/scripts/installers/python.sh",
-                "{{template_dir}}/scripts/installers/graalvm.sh"
```

Comment:
Can u push this one to line and block 255.

Reply:
Hi @freddy123098 , thanks for reviewing and approving. I can't seem to fully understand what needs to be _blocked_ here? Could you please elaborate it again for me? Thanks!

Reply:
Nah it is ok @msmygit it happens.

Reply:
Glad to help u and the others.

Reply:
Can u push the red lined area to the upper part or the next line to keep it corrected and in green.

Reply:
@freddy123098, thank you, but since we need to add the new entry to the list, we need to suffix the already existing last line with a comma character (`,`) in order to achieve the same and hence you see that line in red color as being modified. The last entry in that list will not have the comma character (`,`) at the end.

```
,
                "{{template_dir}}/scripts/installers/cassandra.sh"
```

Did that help clarify?